code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -98,6 +98,33 @@
 			user << "<span class='warning'>You need a working light!</span>"
 			return
 
+	if(istype(W, /obj/item/weapon/storage/box/lights))
+		var/obj/item/weapon/storage/box/lights/B = W
+		var/tmp/replacement_status = 1 // 0-useable lights found, 1-no useable lights, 2-light replacer was full
+
+		if(src.uses == max_uses)
+			if(!B.contents.len) // empty box
+				replacement_status = 1

```

Comment:
I don't think we should pollute the symbol space for stuff that's specific to the body of a single proc.